stroeer/page/article/v1/article_page.proto

```diff
@@ -23,6 +24,10 @@ message ArticlePage {
   // stages for article pages contain companions, links to topic pages, A-Z Modules etc.
   // configuration.fields contains a `stage_type`, which defines the rendering position
   repeated stroeer.page.stage.v1.Stage stages = 3;
+
+  // List of root references such as section_tree, navigation
+  // Each root references holds a unique type and has nested references
+  repeated stroeer.core.v1.Reference references = 4;
```

Comment:
Ich finde die Idee grundsätzlich gut. Problem: `section_tree` gibt's dann zweimal in ArticlePage:

```
ArticlePage.references
ArticlePage.article.section_tree
```

Reply:
Mhm. Das wird auch vom adapter in den core article geschrieben. Eigentlich ist ein section_tree doch ein Attribut einer page. 

Man könnte vermutlich im core article section_tree durch einen string path ersetzen.
Außerdem müsste man aus Konsitenzgründen auch den section_tree aus einer Section in die SectionPage ziehen.
Dann gäbe es die Möglichkeit die page dynamisch zu ändern.

---- 
Das extrem breaking für den core article und würde sehr viel code aus dem adapter in paperboy rein ziehen. 
Ziemlich teuer.

Dann die erste (offensichtliche) Idee die ich hatte: 
Ein normaler Knoten "menue_tree" im Article und Section Model?

wdyt?